doc/releases/changelog-dev.md

```diff
@@ -6,6 +6,9 @@
 
 <h3>Improvements 🛠</h3>
 
+* Converted current tests that used `default.mixed` to use other equivalent devices in-place.
+  [(#6684)](https://github.com/PennyLaneAI/pennylane/pull/6684)
+
```

Comment:
There should also be a changelog entry under either the "New features" or "Improvements" section about porting `default.mixed` to the new device API. I will let @isaacdevlugt pick which section makes more sense.

Reply:
@isaacdevlugt

Reply:
Aaaaaaah!!! Sorry I missed this :notlikethis: